test_verbosefilter.py

```diff
@@ -0,0 +1,108 @@
+# -*- python -*-
+
+"""
+Unit tests for VerboseFilter, a logging helper in adept_openstack.py
+"""
+
+import logging
+import sys
+from mock import Mock
+from unittest2 import TestCase, main
+
+sys.modules['virtualenv'] = Mock()      # Needed for Ed's Fedora system
+
+sys.path.insert(0, 'kommandir/bin')
+
+
+class MyHandler(logging.Handler):
+    """
+    Output handler for logging messages. No output; just push onto list.
+    """
+    def __init__(self, message_list):
+        logging.Handler.__init__(self)
+        self.message_list = message_list
```

Comment:
pylint may get mad (at some point) since there's no ``super`` call here.  The tests run under ubuntu, so god knows what the difference will be compared to Fedora.